From 508e5801df0305917d053525690f00290306af0e Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 5 Sep 2014 19:49:55 -0400
Subject: [ethernet] bnx2x: Remove redundant #ifdef

Message-id: <1409946615-5862-51-git-send-email-mschmidt@redhat.com>
Patchwork-id: 91589
O-Subject: [RHEL7.1 PATCH 50/70] net: ethernet: broadcom: bnx2x: Remove redundant #ifdef
Bugzilla: 1089402
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1089402

Upstream commit:
commit 7d149c5268d78d740cfdb20834328975251e7388
Author: Rasmus Villemoes <linux@rasmusvillemoes.dk>
Date:   Wed Aug 20 15:14:49 2014 +0200

    net: ethernet: broadcom: bnx2x: Remove redundant #ifdef

    Nothing defines _ASM_GENERIC_INT_L64_H, it is a weird way to check for
    64 bit longs, and u64 should be printed using %llx anyway.

    Signed-off-by: Rasmus Villemoes <linux@rasmusvillemoes.dk>
    Signed-off-by: David S. Miller <davem@davemloft.net>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_cmn.c b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_cmn.c
index 076acf1..a41e8fe 100644
--- a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_cmn.c
+++ b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_cmn.c
@@ -483,11 +483,7 @@ static void bnx2x_tpa_start(struct bnx2x_fastpath *fp, u16 queue,
 
 #ifdef BNX2X_STOP_ON_ERROR
  fp->tpa_queue_used |= (1 << queue);
-#ifdef _ASM_GENERIC_INT_L64_H
- DP(NETIF_MSG_RX_STATUS, "fp->tpa_queue_used = 0x%lx\n",
-#else
  DP(NETIF_MSG_RX_STATUS, "fp->tpa_queue_used = 0x%llx\n",
-#endif
     fp->tpa_queue_used);
 #endif
 }
-- 
1.7.1